### ISSUE TITLE
How to add weight decay for the training process?

### ISSUE THREAD

**Issue by Kongsea** [fizyr/keras-retinanet#137]
Have you considered to add weight decay?
How to add it for the training process?
Thank you.

**Comment by hgaiser** [comment]
I've tried it with SGD, but it was a while back. @yhenon was trying this not so long ago, maybe he can share his experiences.

ps. this isn't really an issue, maybe pick this up further on Slack (`#keras-retinanet` channel).
